.github/workflows/release.yml

```diff
@@ -1,60 +1,110 @@
-#         with:
-#           publish: true
-#       - name: Get the short tag
-#         id: get_tag_name
-#         run: |
-#           short_tag=$(echo ${{ steps.release-drafter.outputs.tag_name }} | cut -d. -f1)
-#           echo "SHORT_TAG=$short_tag" >> $GITHUB_OUTPUT
-#   create_npm_release:
-#     needs: create_github_release
-#     runs-on: ubuntu-latest
-#     permissions:
-#       packages: write
-#     env:
-#       REGISTRY: ghcr.io
-#       IMAGE_NAME: ${{ github.repository }}
-#     steps:
-#       - uses: actions/checkout@8459bc0 # v4
-#       - uses: actions/setup-node@c2ac33f # v4, Setup .npmrc file to publish to npm
-#         with:
-#           node-version: '18.x'
-#           registry-url: 'https://registry.npmjs.org'
-#       - run: npm ci
-#       - run: npm publish --access=public
-#         env:
-#           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+jobs:
+  update_version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Run Release Drafter
+        uses: release-drafter/release-drafter@v6
+        id: release-drafter
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Extract Version
+        id: extract_version
+        run: |
+          version="${{ steps.release-drafter.outputs.tag_name }}"
+          version=$(echo "$version" || sed 's/^v//')
+          if [ -z "$version"]; then
+            echo "Version is empty. Exiting"
+            exit 1
+          fi
+          echo "$version" > version.txt
+          echo ::set-output name=version::$version
+      - name: Update package.json and package-lock.json version
+        run: 
+          VERSION=${{ steps.extract_version.outputs.version }}
+          if [ -z "$version"]; then
+            echo "Version is empty. Exiting"
+            exit 1
+          fi
+          jq --arg version "$VERSION" '.version = $version' package.json > package_tmp.json && mv package_tmp.json package.json
+          jq --arg version "$VERSION" '.version = $version' package-lock.json > package_lock_tmp.json && mv package_lock_tmp.json package-lock.json
+      - name: Update docusaurus.config.js version 
+        run: |
+          VERSION=${{ steps.extract_version.outputs.version }}
+          if [ -z "$version"]; then
+            echo "Version is empty. Exiting"
+            exit 1
+          fi
+          sed -i "s/version: '.*'/version: '$VERSION'/" website/docusaurus.config.js
+      - name: Commit version change
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action"
+          git add package.json package-lock.json website/docusaurus.config.js
+          git commit -m "chore: update version to ${{ steps.extract_version.outputs.version }}"
+  
+      - name: Push Changes
+        uses: ad-m/github-push-action@master
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@divinetettey - we are super close now. These changes are perfect. Can we make sure that this commit and push won't re-trigger another release? Every commit to `main`, according to this workflow will result in a release. That is spot on, apart from we don't want this commit to trigger another re-deploy.

Reply:
what about adding a phrase to the commit message of the version bump  eg. chore(skip ci) and skip the workflow if the commit message contains the phrase ?

Reply:
Yes, perfect! GitHub Actions detects the usage of [[skip ci]](https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs#:~:text=a%20pull%20request%3A-,%5Bskip%20ci%5D,-%5Bci%20skip%5D).